src/statsig/index.ts

```diff
@@ -89,14 +88,8 @@ function _getDynamicConfigParams<T extends Record<string, StatsigParameter>>({
 }
 
 export function getMultichainFeatures() {
-  const multichainParams = _getDynamicConfigParams(
-    DynamicConfigs[StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES]
-  )
-  const filteredParams = {} as { [key: string]: NetworkId[] }
-  Object.entries(multichainParams).forEach(([key, value]) => {
-    filteredParams[key] = value.filter((networkId) => networkId in NetworkId)
-  })
-  return filteredParams
+  // just use defaults, which is celo
+  return DynamicConfigs[StatsigMultiNetworkDynamicConfig.MULTI_CHAIN_FEATURES].defaultValues
```

Comment:
this could be cleaned up better, but leaving this as is for now as we might clean up this dyn config altogether in Valora